deploy/contributor/production/.env.production

```
@@ -0,0 +1,4 @@
+# Contributor service production environment
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)